.pre-commit-config.yaml

```diff
@@ -6,6 +6,7 @@ repos:
       - id: check-merge-conflict
       - id: check-case-conflict
       - id: check-yaml
+        args: ["--unsafe"]
```

Comment:
Required such that it only checks for syntax and does not try to evaluate the yaml: https://stackoverflow.com/questions/59413979/how-exclude-ref-tag-from-check-yaml-git-hook

For the evaluation we have Alexeys tests anyways. Without this, linting will not pass as it does not understand the code below in the mkdocs.yml, which we need for the icons.

```
  - pymdownx.emoji:
      emoji_index: !!python/name:material.extensions.emoji.twemoji
      emoji_generator: !!python/name:material.extensions.emoji.to_svg
```